MyHeartCounts/MyHeartCounts.swift

```diff
@@ -32,5 +32,6 @@ struct MyHeartCounts: App {
             )
         }
         .environment(appDelegate)
+        .environment(StudyDefinitionLoader.shared)
```

Comment:
What is the need for this singleton; couldn't we rely on the Spezi Module injection mechanism here?